README.md

```diff
@@ -335,7 +335,7 @@ A value that implements the ChainRec specification must also implement the Chain
 #### `chainRec` method
 
 ```hs
-chainRec :: ChainRec m => ((a -> c) -> (b -> c) -> a -> m c) -> a -> m b
+chainRec :: ChainRec m => ((((a -> c), (b -> c), a) -> m c), a) -> m b
```

Comment:
Or should it be `chainRec :: ChainRec m => ((a -> c, b -> c, a) -> m c, a) -> m b`?